framework/db/src/main/java/com/cloud/utils/db/TransactionMBeanImpl.java

```diff
@@ -45,14 +48,15 @@ public void removeTransaction(TransactionLegacy txn) {
 
```

Comment:
I would expect a decrement-call in the `removeTransaction(..)`

Reply:
Fixed/removed code